<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPersone</class>
 <widget class="QWidget" name="NewPersone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="teJob">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teBiography">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>166</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Основне місце роботи</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата народження</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Прізвище</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leLastName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ім'я</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFirstName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>По-батькові</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leMiddleName"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pbSubmit">
      <property name="text">
       <string>Підтвердити</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbCancel">
      <property name="text">
       <string>Відмінити</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Біографія</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Батько</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Мати</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>teJob</zorder>
  <zorder>teBiography</zorder>
  <zorder>label_4</zorder>
  <zorder>dateEdit</zorder>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>comboBox</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
